playroom/snippets.tsx

```diff
@@ -890,6 +890,80 @@ const exampleScreens: Array<Snippet> = [
     },
 ];
 
+const navigationBarSnippets = [
+    {
+        group: 'NavigationBar',
+        name: 'MainNavigationBar',
+        code: `
+<MainNavigationBar
+  sections={["Start", "Account", "Explore", "Support"].map((title, idx) => ({
+    title,
+    onPress: () => setState("index", idx),
+  }))}
+  selectedIndex={getState("index", 0)}
+  right={
+    <NavigationBarActionGroup>
+      <NavigationBarAction
+        onPress={() => {}}
+        aria-label="shopping cart with 2 items"
+      >
+        <Badge value={2}>
+          <IconShoppingCartRegular color="currentColor" />
+        </Badge>
+      </NavigationBarAction>
+      <NavigationBarAction onPress={() => {}} aria-label="Open profile">
+        <Circle
+          backgroundImage="https://i.pravatar.cc/100?img=31"
+          size={32}
```

Comment:
I can see in specs, in mobile view the size of the profile photo is 24. Can you confirm this @HectorSancho

Reply:
Could it be interesting for the future to componentize some of these actions (UserProfile, shoppingCart, help...) in order to release developers from having to know how to configure sizes, colors etc, and make changes easier?

Reply:
changed

Reply:
>Could it be interesting for the future to componentize some of these actions (UserProfile, shoppingCart, help...) in order to release developers from having to know how to configure sizes, colors etc, and make changes easier?

If we find this is a repetitive pattern, for sure. But I don't think so